python/lsst/afw/image/apCorrMap/_apCorrMap.cc

```diff
@@ -67,7 +67,7 @@ void wrapApCorrMap(lsst::utils::python::WrapperCollection &wrappers) {
     });
 }
 
-PYBIND11_MODULE(apCorrMap, mod) {
+PYBIND11_MODULE(_apCorrMap, mod) {
```

Comment:
I had forgotten that we had these separate subpackage builds in `afw.image` with their own `SConscript`s inside directories (`apCorrMap`, `exposure`, `image`, `maskedImage`, `pixel`).  With the exception of `pixel`, these are not intended to appear as public namespaces in Python (they aren't namespaces in C++), and I don't think there's any reason for them to be separate anymore.  Could you merge those packages (`apCorrMap`, `exposure`, `image`, `maskedImage`) into the `_imageLib` build and take them out of their subdirectories?

Reply:
I has some dependency problems when doing this. I would push this back and revisit when all other sub packages are using the pybind wrappers.